ocs_ci/ocs/acm/acm.py

```diff
@@ -566,16 +566,16 @@ def login_to_acm():
     driver = login_ui(url)
     page_nav = AcmPageNavigator()
     page_nav.page_has_loaded(retries=10, sleep_time=5)
-    look_for_local_cluster = page_nav.wait_until_expected_text_is_found(
+    look_for_administrator_dropdown = page_nav.wait_until_expected_text_is_found(
```

Comment:
Hi Aman, 
I checked our ACM setup with OCP 4.20 and MCE 2.9.0, and local-cluster is still there.
Please work out two scenarios so that, in case local-cluster does not work, an Administrator is checked.

<img width="859" height="1004" alt="Image" src="https://github.com/user-attachments/assets/b2dde7ff-a5a1-4ea6-9612-2430b4c700bf" />